server/server.go

```diff
@@ -43,22 +50,24 @@ func Start(ip string, port int, cert string, key string, debug bool, t *Tracker,
 		torStr = "Tor"
 	}
 
-	fmt.Printf("%sShuffle Listening on TCP %s:%d (pool size: %d)\n", torStr, ip, port, t.poolSize)
+	log.Infof("%sShuffle Listening on TCP %s:%d (pool size: %d)\n", torStr, ip, port, t.poolSize)
```

Comment:
Think we are missing `logListener` here.

Reply:
Sorry + Thanks!

Reply:
24d569e